### PR TITLE
Make Calling Sample textfield validate the Teams user id format

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What

Make Calling Sample textfield validate the Teams user id format

image

# Why

QoL improvement based on TDL bug https://skype.visualstudio.com/SPOOL/_workitems/edit/3520928

# How Tested
Locally (see above screenshot)

### CODE REVIEW

File: samples/Calling/src/app/views/HomeScreen.tsx [hunk 1]
```diff
@@ -232,7 +236,21 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                     label={teamsIdLabel}
                     required
                     placeholder={'Enter a Teams id'}
```
Comment (mgamis-msft) [inline samples/Calling/src/app/views/HomeScreen.tsx:238]: Maybe we can also change the placeholder to 'Enter your Teams id (8:orgid:<UUID>)'

File: samples/Calling/src/app/views/HomeScreen.tsx [hunk 2]
```diff
@@ -284,10 +302,21 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
                     label={'Teams user ID'}
                     required
                     placeholder={'Enter a Teams user ID'}
```
Comment (mgamis-msft) [inline samples/Calling/src/app/views/HomeScreen.tsx:304]: Maybe we can also change the placeholder to 'Enter a Teams id (8:orgid:<UUID>)'

**Review by mgamis-msft — APPROVED** [review]

File: samples/Calling/src/app/views/HomeScreen.tsx (hunk 1 above)
Comment (JamesBurnside) [reply samples/Calling/src/app/views/HomeScreen.tsx:238]: Good idea I'll add that

Comment (JamesBurnside) [reply samples/Calling/src/app/views/HomeScreen.tsx]: updated

File: samples/Calling/src/app/views/HomeScreen.tsx (hunk 2 above)
Comment (JamesBurnside) [reply samples/Calling/src/app/views/HomeScreen.tsx]: updated

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**